packages/python/plotly/plotly/express/_core.py

```diff
@@ -1,25 +1,27 @@
 import plotly.graph_objs as go
 import plotly.io as pio
 from collections import namedtuple, OrderedDict
+from collections.abc import Sequence
 from ._special_inputs import IdentityMap, Constant, Range
 from .trendline_functions import ols, lowess, rolling, expanding, ewm
 
 from _plotly_utils.basevalidators import ColorscaleValidator
 from plotly.colors import qualitative, sequential
 import math
-from packaging import version
-import pandas as pd
-import numpy as np
+
+import narwhals.stable.v1 as nw
+from narwhals.dependencies import is_into_series
```

Comment:
@FBruzzesi Should these second two import statements also import from the stable namespace? I would expect the import format to be consistent, but not sure if there is some reasoning I'm missing.

Reply:
Fixed that, yet `generate_unique_token` is yet not exposed. We have an open issue for it